From f1943b1982baad201f452bfcd35c56368321ae42 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 22 Aug 2017 16:56:40 +0300
Subject: [PATCH 1183/1345] fix: dts: arm64: marvell: Remove Comphy handlers
 from PCIe nodes

commit  893bda09593348f0f36c8b42cff44337e658a256 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Comphy is disabled in vanilla device trees, comphy entries
should be removed from PCIe nodes.
Patch fix broken PCIe on vanilla device trees.

Change-Id: Iee2961fe14a6e2ca8ed455f86d451db4edd05ad8
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43315
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/marvell/armada-7040-db-pp2-vanilla.dts     |    2 --
 .../dts/marvell/armada-8040-db-pp2-vanilla.dts     |   10 ----------
 .../dts/marvell/armada-8040-mcbin-pp2-vanilla.dts  |    6 ------
 3 files changed, 18 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
index 046b131..ac0e02f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
@@ -202,8 +202,6 @@
 
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
index a6ee25b..cc9610f 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
@@ -179,13 +179,9 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 	cpn-110-slave {
@@ -272,18 +268,12 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy1 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "okay";
-			phys = <&comphy1 4 COMPHY_PCIE1>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy1 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
index f3021d2..9fa94f6 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
@@ -206,12 +206,6 @@
 		pcie@0x600000 {
 			num-lanes = <4>;
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0
-				&comphy0 1 COMPHY_PCIE0
-				&comphy0 2 COMPHY_PCIE0
-				&comphy0 3 COMPHY_PCIE0
-				>;
-			phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 			reset-gpios = <&cpm_gpio1 20 GPIO_ACTIVE_HIGH>;
 			pinctrl-names = "default";
 			pinctrl-0 = <&pcie_reset_pins>;
-- 
1.7.9.5

